### ISSUE TITLE
My minecraft doesnt launch and it says that it is the dream block not rendenring 

### ISSUE THREAD

**Issue by loupcraft** [MayaqqDev/Estrogen#227]
[crash-2024-12-19_13.50.32-client.txt](https://github.com/user-attachments/files/18198900/crash-2024-12-19_13.50.32-client.txt)
i hope this helps : )

**Comment by MayaqqDev** [comment]
is this still an issue?

**Comment by Jo-se-P-H** [comment]
Yes, still and issue, worked 5 mins ago not even sure what i changed
[crash-2025-01-08_19.57.16-client.txt](https://github.com/user-attachments/files/18355677/crash-2025-01-08_19.57.16-client.txt)
update: crash seems to be caused by an unrelated mod called "Nether, ender and more"(the file is called more minerals)( https://www.curseforge.com/minecraft/mc-mods/mcminerals ), after removing everything works again

**Comment by MayaqqDev** [comment]
Wow thats awesome, a closed source random mod that has nothing to do with rendering :sob:

**Comment by MayaqqDev** [comment]
Oh, obviously its fucking mcreator

**Comment by MayaqqDev** [comment]
Thank you for letting me stumble upon the most cursed shit I have saw so far.

**Comment by MayaqqDev** [comment]
This mod, supposedly adding ores is:
- Mcreator closed source
- For some reason has ALl the colors for damn teleporters and dimensions??
- Okay it just has what the hell even is this what the hell, why does this add like billion different items, they dont even make sense, the description doesnt explain it at all, IT ADDS FUCKING BIOMES AND ENTITIES ???? WHAT THE FUCK IS BLUE TNT
